rascal2/ui/presenter.py

```diff
@@ -177,6 +178,20 @@ def handle_event(self):
         elif isinstance(event, LogData):
             self.view.logging.log(event.level, event.msg)
 
+    def edit_project(self, calculation: Calculations, model: LayerModels, geometry: Geometries) -> None:
+        """Updates the project parameters.
+
+        Parameters
+        ----------
+        calculation : Calculations
+            The updated calculation of the project.
+        model : LayerModels
+            The updated model of the project.
+        geometry : Geometries
+            The updated geometry type of the project.
+        """
+        self.model.update_project_general_settings(calculation, model, geometry)
+
```

Comment:
as above, we probably want this to take all the project attributes as a dict when we save and process them in a batch

Reply:
@StephenNneji no: all the RATapi update added was the ability to use dicts *inside* the ClassList for each field. dict unpacking like we're doing here is just standard Python, e.g.
```python
from pydantic import BaseModel

class Model(BaseModel):
    var1: int
    var2: str

my_vars = {'var1': 3, 'var2': 'Hello World!'}
Model(**my_vars)
```
in this case the content of the fields will be the actual ClassLists of parameters etc. that we are expecting, so none of the new features of RascalSoftware/python-RAT#79 are required.